pages/index.js

```diff
@@ -1,3 +1,34 @@
+import styled from "styled-components";
+import movieData from "../db.json";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  return (
+    <main>
+      <h1>Your Movies</h1>
+      <br />
```

Comment:
I would recommend to use styled-components/CSS for spacing instead of the `br` element.
```suggestion

```